torch_xla/csrc/translator.h

```diff
@@ -25,6 +25,8 @@ struct XlaTranslationResult {
   XlaComputationInOut::SizeOpValues ret_size_op_values;
 };
 
+class ComputationContext;
```

Comment:
This does not need to be on the interface. Can be within the anon namespace of the cpp file.

Reply:
Would you rather have `translation_handlers_` as a global variable instead of static field? Because this is the reason I needed this forward declaration.

Reply:
I think so. No?

Reply:
Ok, done.